src/main/java/yappingbot/stringconstants/ReplyTextMessages.java

```diff
@@ -0,0 +1,98 @@
+package yappingbot.stringconstants;
+
+/**
+ * String constants of messages that the bot can use to reply.
+ */
+public final class ReplyTextMessages {
```

Comment:
I like the thought of putting all the string constants that the bot can use to reply in a class 👍